test/Microsoft.AspNetCore.Razor.Test.Common/Language/TestProject.cs

```diff
@@ -3,23 +3,41 @@
 
 using System;
 using System.IO;
-using System.Reflection;
 
 namespace Microsoft.AspNetCore.Razor.Language
 {
     public static class TestProject
     {
         public static string GetProjectDirectory(Type type)
```

Comment:
Can we use `TestPathUtilities.GetSolutionRootDirectory`

Reply:
Is there an API we could use that doesn't have utilities in the name? :trollface:

Reply:
Didn't know that existed, sounds like a great alternative!